pages/items.vue

```diff
@@ -49,4 +49,10 @@ export default {
     }
   }
 }
+p {
+  color: black;
+  border: solid 3px #00a5b1;
```

Comment:
style/color.scssに定義してあるから
```suggestion
  border: solid 3px $mainColor1;
```
でいいと思う